packages/dialtone-vue2/components/input/input.vue

```diff
@@ -15,6 +16,7 @@
           ref="label"
           data-qa="dt-input-label"
           :class="[
+            'd-input__label-text',
```

Comment:
@juliodialpad this is an actual class with a lot of styling in `input.less`. Did we remove this on purpose? If so we should also remove it from `input.less`. You can check the percy to see the styling diff.

Reply:
d-input__description as well.

Reply:
Did I remove that? I can't find that line removed on my PR, certainly not intended by me

Reply:
yeah this line:
https://github.com/dialpad/dialtone/pull/392/files#diff-eada107db256f0591fc3a4fa8c6d065ba54583930249bf3c6a28b9240f55c7a7L20

and this line:
https://github.com/dialpad/dialtone/pull/392/files#diff-eada107db256f0591fc3a4fa8c6d065ba54583930249bf3c6a28b9240f55c7a7L33

Reply:
Oh I see, removed all `base-input*` because I couldn't find any reference of style on `input.less` I can't find any reference now either on https://github.com/dialpad/dialtone/blob/staging/packages/dialtone-css/lib/build/less/components/input.less

Reply:
base-input__label-text is not there but d-input__label-text is.. I'm actually really confused where this actually came from.

https://github.com/dialpad/dialtone/pull/21/files

Why did we add styles here, but then not apply them to the actual input component?

Reply:
I think it was due to the confusion between `base-input*` and `d-input*` 🤔 but yeah, those weren't being applied to the component actually

Reply:
Alright found this conversation from ages ago.. looks like these classes were just supposed to be deleted.

https://github.com/dialpad/dialtone-vue/pull/1314/files#r1411122015

Reply:
Nice find! yeah, seems like we forgot to remove it 😅

Reply:
Okay looks good now, percy passing again.